.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,89 @@
+name: Bug Report
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: |
+        <p align="center"><em>Thank you for filing a bug report!</em></p>
+
+        > [!IMPORTANT]
+        > - Please provide descriptive titles. Good titles help us address your report faster.
+        > - Please search existing issues to avoid creating duplicates.
+        > - For enhancement requests, please use GitHub Discussions.
+        > - Please fill out the template below to the best of your ability.
+        > - Please describe the issue as much as possible and provide steps to reproduce it screenshots if possible.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Describe the bug clearly and concisely.
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Please write a description of what was supposed to happen.
+    validations:
+      required: false
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: Please write a description of what actually happened.
+    validations:
+      required: false
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Please Provide numbered steps to reproduce this issue so developers can replicate them easily.
+    validations:
+      required: false
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: |
+        If applicable, add screenshots to help explain your problem.
+  - type: dropdown
+    id: component
+    attributes:
+      label: What component is this issue related to?
+      options:
+        - Alerts
+        - API
+        - Caching
+        - Citator
+        - Continuous Integration
+        - Data Quality
+        - Database
+        - Design
+        - Documentation
+        - HTML
+        - Javascript
+        - Judge DB
+        - Oral Arguments
+        - Performance
+        - "@recap.email"
+        - Search
+        - Security
+        - Tests
+        - UX
+        - Visualizations
+        - Webhook
+    validations:
+      required: true
+  - type: dropdown
+    id: severity
+    attributes:
+      label: What is the severity of the bug?
+      options:
+        - Trivial
+        - Low
+        - Medium
+        - High
+        - Critical - only if completely down
```

The feedback we've had is to make severity have a TBD option and to remove the components piece:

```suggestion
  - type: dropdown
    id: severity
    attributes:
      label: What is the severity of the bug?
      options:
        - TBD
        - Trivial
        - Low
        - Medium
        - High
        - Critical - only if completely down
```

This looks great though and I think it'll help!